### PR TITLE
test: add CloudFront e2e test

### PR DESCRIPTION
**Pull request by iamhopaul123**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: e2e/cloudfront/cloudfront_test.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+	})
+
+	Context("when adding new environment", func() {
+		var err error
+		BeforeAll(func() {
+			_, err = cli.EnvInit(&client.EnvInitRequest{
+				AppName: appName,
+				EnvName: "test",
+				Profile: "default",
+				Prod:    false,
```
Comment (dannyrandall) [inline e2e/cloudfront/cloudfront_test.go]:
```suggestion
```

File: e2e/cloudfront/cloudfront_test.go [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+			Expect(appShowOutput.URI).To(Equal(domainName))
+		})
+	})
+
+	Context("when adding new environment", func() {
+		var err error
+		BeforeAll(func() {
+			_, err = cli.EnvInit(&client.EnvInitRequest{
+				AppName: appName,
+				EnvName: "test",
```
Comment (dannyrandall) [inline e2e/cloudfront/cloudfront_test.go:54]: i'm probably dumb but how does cloudfront get enabled 🥺

Comment (paragbhingre) [reply e2e/cloudfront/cloudfront_test.go:54]: +1, I was also searching for the manifest or code that adds `cdn: true` snippet to the env manifest.

Comment (iamhopaul123) [reply e2e/cloudfront/cloudfront_test.go:54]: Oh I forgot to include the manifest files

File: e2e/cloudfront/copilot/frontend/manifest.yml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+type: Load Balanced Web Service
+image:
+  build: 
+    dockerfile: src/Dockerfile
+    context: src
+  port: 80
+
+http:
+  path: '/'
+  alias: ["cloudfront.copilot-e2e-tests.ecs.aws.dev"]
```
Comment (paragbhingre) [inline e2e/cloudfront/copilot/frontend/manifest.yml:12]: Does copilot creates A record for this alias when we deploy the service?

**Review by paragbhingre — APPROVED** [review]

File: e2e/cloudfront/copilot/frontend/manifest.yml (hunk 3 above)
Comment (iamhopaul123) [reply e2e/cloudfront/copilot/frontend/manifest.yml:12]: Yes it does.

**Review by huanjani — APPROVED** [review]
LGTM!

**Pull request merged**